UI API Acceptance tests are failing after introducing breaking change in UI API

**Description**

UI API Acceptance tests were not fixed after introducing breaking change in UI API related to Service Binding statistics (https://github.com/kyma-project/kyma/pull/969).

```
'test-core-core-ui-api-acceptance' has Failed status
21:27:59 --- FAIL: TestServiceInstanceMutationsAndQueries (0.60s)
21:27:59     serviceinstance_test.go:91: Subscribe instances
21:27:59     serviceinstance_test.go:95: Create instance
21:27:59     assertions.go:247: 
    Error Trace:    serviceinstance_test.go:98
21:27:59             Error:          Received unexpected error:
21:27:59                             graphql: Cannot query field "name" on type "ServiceBindings".
21:27:59             Test:           TestServiceInstanceMutationsAndQueries
21:27:59 FAIL
```

--- a/tests/ui-api-layer-acceptance-tests/domain/servicecatalog/servicebinding_test.go
+++ b/tests/ui-api-layer-acceptance-tests/domain/servicecatalog/servicebinding_test.go
@@ -129,22 +129,24 @@ func TestServiceBindingMutationsAndQueries(t *testing.T) {
 	instanceRes, err := querySingleInstance(c, `
 		name
 		serviceBindings {
-			name
-			serviceInstanceName
-			environment
-			secret {
+			serviceBindings {
 				name
+				serviceInstanceName
 				environment
-				data
-			}
-			status {
-				type
+				secret {
+					name
+					environment
+					data
+				}
+				status {
+					type
+				}
 			}
 		}
 	`, instance)
 
 	assert.NoError(t, err)
-	assertBindingExistsAndEqual(t, binding, instanceRes.ServiceInstance.ServiceBindings)
+	assertBindingExistsAndEqual(t, binding, instanceRes.ServiceInstance.ServiceBindings.ServiceBindings)
 
 	t.Log("Delete Binding")
 	deleteRes, err := deleteBinding(c, deleteBindingOutput)

--- a/tests/ui-api-layer-acceptance-tests/domain/servicecatalog/serviceinstance_test.go
+++ b/tests/ui-api-layer-acceptance-tests/domain/servicecatalog/serviceinstance_test.go
@@ -47,9 +47,13 @@ type ServiceInstance struct {
 	CreationTimestamp    int
 	Labels               []string
 	Status               ServiceInstanceStatus
-	ServiceBindings      []ServiceBinding
+	ServiceBindings      ServiceBindings
 	ServiceBindingUsages []ServiceBindingUsage
 	Bindable             bool
+}
+
+type ServiceBindings struct {
+	ServiceBindings []ServiceBinding
 }
 
 type ServiceInstanceStatus struct {
@@ -336,15 +340,23 @@ func instanceDetailsFields() string {
 			activated
 		}
 		serviceBindings {
-			name
-			serviceInstanceName
-			environment
-			secret {
+			serviceBindings {
 				name
+				serviceInstanceName
 				environment
-				data
+				secret {
+					name
+					environment
+					data
+				}
 			}
-		}
+			stats {
+				ready
+				failed
+				pending
+				unknown
+			}
+        }
 		serviceBindingUsages {
 			name
 			environment

